changelog.d/309.feature

```diff
@@ -0,0 +1 @@
+Add support for 'push_type' option in app config. This is needed if notifications are to be silenced client side.
```

Comment:
```suggestion
Add a new `push_type` configuration option for APNs apps, to control the value of the `apns-push-type` header when sending requests.
```

Reply:
Also feel free to credit yourself by adding a `Contributed by ...` (with your name, username or anything else) at the end if you wish to do so :)